controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -208,7 +209,7 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageRequest(
 
 	rbdStorageRequest := &metav1.PartialObjectMetadata{}
 	rbdStorageRequest.SetGroupVersionKind(ocsv1alpha1.GroupVersion.WithKind("StorageRequest"))
-	rbdStorageRequest.Name = getStorageRequestName(
+	rbdStorageRequest.Name = util.GetStorageRequestName(
```

Comment:
Where do we need this outside of the current package?

Reply:
Since I am moving the filling of the method to util, I need to export this or create a copy of the code